uint8 role
uint8 id
uint32 systemTime
vector3d_t position
vector3d_t eop
vector3d_t velocity
float32[8] distances
vector3d_t imuGyro
vector3d_t imuAcc
vector3d_t angle
float32[4] quaternions
float32 supplyVoltage
node2_t[] nodes
